deploy/kubernetes/service-broker.yml

```diff
@@ -5,10 +5,10 @@ metadata:
   labels:
     run: bookstore-broker
 spec:
-  url: http://bookstore-broker.bookstore-broker.svc.cluster.local
+  url: http://35.197.249.208
```

Comment:
Will the DNS name not work with `LoadBalanced`? This IP address won't be valid for all deployments, will it?

Reply:
Yes using the internal DNS works for me